src/inventory_ui.cpp

```diff
@@ -23,7 +23,11 @@
 #include <algorithm>
 
 /** The minimal gap between two cells */
-static const int min_cell_gap = 1;
+static const int min_cell_gap = 2;
+/** The gap between two cells when screen space is limited*/
+static const int normal_cell_gap = 4;
+/** The minimal gap between the first cell and denial */
+static const int min_denial_gap = 1;
```

Comment:
> Does this result in output of the form item_denial where _ is a single space?

Yes, nothing's changed in that respect except for the named constant. I'm bumping it since denials now can start with capital letters.